### ISSUE TITLE
Cargo build failure on M1 mac

### ISSUE THREAD

**Issue by tpmoney** [dankamongmen/libnotcurses-sys#22]
With a new application (`cargo init hello_world`), and `notcurses = 3.0.1` as a dependency in cargo.toml, and `notcurses` 3.0.8 installed via homebrew, running `cargo build` produced the following error:

```
Compiling libnotcurses-sys v3.6.0
error[E0308]: mismatched types
    --> $HOME/.cargo/registry/src/github.com-1ecc6299db9ec823/libnotcurses-sys-3.6.0/src/plane/methods.rs:1610:59
     |
1610 |             unsafe { c_api::ncpile_render_to_buffer(self, &mut buf, &mut len.try_into().unwrap()) },
     |                                                           ^^^^^^^^ expected `i8`, found `u8`
     |
     = note:    expected raw pointer `*mut *mut i8`
             found mutable reference `&mut *mut u8`
```

**Comment by tpmoney** [comment]
That seems to build OK for me.

**Comment by joseluis** [comment]
That's great. I've just published notcurses v3.0.2 to crates.io.
